services/mapper.go

```diff
@@ -84,15 +156,9 @@ func FeeOps(tx *evmClient.LoadedTransaction) []*RosettaTypes.Operation {
 		return ops
 	}
 
-	// in internal is 
-	// burntOp := &RosettaTypes.Operation{
-	// 	OperationIdentifier: &RosettaTypes.OperationIdentifier{
-	// 		Index: 0, // nolint:gomnd
-	// 	},
-	idx := len(ops)
 	burntOp := &RosettaTypes.Operation{
 		OperationIdentifier: &RosettaTypes.OperationIdentifier{
-			Index: int64(idx), // nolint:gomnd
+			Index: 0, // nolint:gomnd
```

Comment:
@ash-krishnan @xiaying-peng to confirm this change

Reply:
this is related to support whitelist, @ash-krishnan will have more context